internal/task/scripts/balance.go

```diff
@@ -0,0 +1,47 @@
+/*
+ *  Copyright (c) 2021 NetEase Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+/*
+ * Project: CurveAdm
+ * Created Date: 2022-04-22
+ * Author: Ye Tao (YhhHaa)
+ */
+
+package scripts
+
+// part of deploying used to balance curvebs cluster
+var BALANCE = `
+#!/bin/bash
+
+# a. define balance operation
+balance() {
+	# get logic pool list
+	targets=$(curve_ops_tool logical-pool-list | grep -o 'id = [0-9]*')
+
+	# balance curvebs leader
+	for val in ${targets}
+	do
+		id=$(echo $val | grep -o '[0-9]*')
+		if [ -n "$id" ]; then
+			curve_ops_tool rapid-leader-schedule -logical_pool_id=${id}
+		fi
+	done
+}
+
+# b. balance
+balance
```

Comment:
actually you should execute `curve_ops_tool rapid-leader-schedule` once, and then check the result, you can refer to the [script](https://github1s.com/opencurve/curve/blob/master/curve-ansible/roles/rapid_leader_schedule/templates/check_leader_range.sh.j2).